### PR TITLE
rbd: misc fixes

### PR DESCRIPTION
**Pull request by lxbsz**

### CODE REVIEW

File: rbd.c [hunk 1]
```diff
@@ -238,9 +236,10 @@ static int tcmu_rbd_service_register(struct tcmu_device *dev)
 }
 
 #ifdef RBD_LOCK_ACQUIRE_SUPPORT
-static void tcmu_rbd_service_status_update(struct tcmu_device *dev,
+static int tcmu_rbd_service_status_update(struct tcmu_device *dev,
 					   bool has_lock)
 {
+	return 0;
```
Comment (idryomov) [inline rbd.c:242]: Fix "rbd: rbd: ..." in the commit title for this change.

File: rbd.c [hunk 2]
```diff
@@ -216,8 +216,6 @@ static int tcmu_rbd_service_register(struct tcmu_device *dev)
 	}
 
 	ret = tcmu_rbd_report_event(dev);
-	if (ret < 0)
-		tcmu_dev_err(dev, "Could not update status. (Err %d)\n", ret);
```
[the remark this answers is not in the document]
Comment (lxbsz) [reply rbd.c:220]: This exists in master, there has already one error log in `tcmu_rbd_report_event() --> tcmu_rbd_service_status_update()`. 
Not introduced by #667, that PR just switched to use  `tcmu_rbd_service_status_update()` directly.

File: rbd.c (hunk 1 above)
Comment (lxbsz) [reply rbd.c:242]: Done. Thanks.

File: rbd.c (hunk 2 above)
[the remark this answers is not in the document]
Comment (idryomov) [reply rbd.c:220]: I see, looks like I got confused there.

**Review by idryomov — APPROVED** [review]

**Comment by lxbsz** [comment]
Merging it, thanks @idryomov

**Pull request merged**